composer/src/main/kotlin/com/gojuno/composer/Apk.kt

```diff
@@ -61,4 +62,7 @@ fun parseTestRunner(testApkPath: String): TestRunner =
                 }
                 .toSingle()
                 .toBlocking()
-                .value()
\ No newline at end of file
+                .value()
+
+fun parseTests(testApkPath: String) : List<String> =
```

Comment:
Also, we need handling of our own annotations to modify resulting junit report. How this can be possible without forking composer? Any plugin mechanism?

Reply:
answering my own question: probably we need something like [this](https://github.com/linkedin/dex-test-parser/issues/9#issuecomment-361446918)

Reply:
Yeah, we discussed plugins before, it shouldn't be super hard since we're running Composer on JVM. We can add a parameter to load plugin(s) as jars and can add abstractions and APIs to hook into

@dsvoronin can you please create a separate issue for Plugin System?

Reply:
#140